lib/util.js

```diff
@@ -27,6 +27,11 @@ function isConflictError(err) {
     return ec.valid && (ec.code === grpc.status.ABORTED || ec.code === grpc.status.FAILED_PRECONDITION);
 }
 exports.isConflictError = isConflictError;
+function isUnauthenticatedError(err) {
+    var ec = errorCode(err);
+    return ec.valid && (ec.code === grpc.status.UNAUTHENTICATED);
```

Comment:
1. These are generated code from *src/util.ts*. But, I will look more into it.
2. Cool! It would be really great to adopt **tsdoc**. Will be bringing this up to the team.